### DISCUSSION TITLE
Help in CI process.

### DISCUSSION THREAD

**Discussion by KuNman** [RealEstateCore/rec#223] [General]
Hey @hammar, following to [this](https://github.com/RealEstateCore/rec/issues/219#issuecomment-1357485051) answer, let me know if you need some help in CI process (or any other topic within this repo), I will be glad to help.

You can find me [here](preempt.tallest-0d@icloud.com).

**Comment by hammar** [comment]
Appreciate the offer, but got it sorted already. 

Cheers,

Karl
